docs/reference/qrng/chains.md

```diff
@@ -93,7 +93,7 @@ beyond one level deep inside and element (i.e. <div>). See /dev/tabs.md
 to not break as well.
 -->
 
-## ANU Mainnet
+## ANU Mainnets Chain ID
```

Comment:
I think this should just be "ANU Mainnets" - the tables have more than just chain IDs (same applies to the other headings).